addons/actions/src/components/ActionLogger/index.tsx

```diff
@@ -1,16 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import Inspector from 'react-inspector';
 
 import { withTheme } from '@storybook/theming';
 import { ActionBar } from '@storybook/components';
 
 import { Actions, Action, Wrapper, InspectorContainer, Counter } from './style';
+import { ActionDisplay } from '../../models';
 
-const ActionLogger = withTheme(({ actions, onClear, theme }) => (
+interface ActionLoggerProps {
+  actions: ActionDisplay[];
+  onClear: (...args: any) => any;
```

Comment:
There's no point in returning something from `onClear` right? So can't we make it it stricter to `void`.